### ISSUE TITLE
'Insert Blockquote' button not working properly with Lists

### ISSUE THREAD

**Issue by TheDemystifier** [Neftas/supplementary-buttons-anki#112]
_Here's my field content :_ 

paragraph
- list-item 1
quote
- list-item 2

_When I select `quote` and click 'Insert Blockquote', I expect it to become :_ 

paragraph
- list-item 1
    > quote
- list-item 2

_But instead it becomes : _ 

paragraph
- list-item 1
    - > quote
- list-item 2

Take a look : 
<img src="http://i.imgur.com/EDCKEJW.png" />

**Comment by Neftas** [comment]
How did you create that first list (your very first example)? I can't really seem to reproduce it using the "create unordered list" button, unless you create two different lists and put the quote in between them and manually indent the quote by putting spaces in front of it.

Unless you used Markdown to create your first example, in which case it's not recommended that you mix it with the other formatting functionality.

If you created your fist example with the formatting buttons (_not_ with Markdown), please post the HTML of that card here.

**Comment by TheDemystifier** [comment]
Here's a <a href="https://www.youtube.com/watch?v=b8Olzf1gpUw">video</a> I recorded showing the exact steps.
and Here's the generated HTML code : 
```html
paragraph<div><ul style="margin-left: 20px; "><li>list-item 1<br /></li></ul><blockquote><ul style="margin-left: 20px; "><li>quote</li></ul></blockquote><ul style="margin-left: 20px; "><li>list-item 2</li></ul></div>
```

**Comment by Neftas** [comment]
@TheDemystifier Thanks for going through the trouble of making a video. I can reproduce the issue and am currently looking into ways to correct this.

**Comment by Neftas** [comment]
@TheDemystifier I've just pushed a new branch with commit 432a7fb. Could you check that one out and test if this solves this issue for you? Thanks!

**Comment by TheDemystifier** [comment]
Yes, just tested it, and it's working properly. Awesome! Thanks!

**Comment by Neftas** [comment]
Great, then it's going to be part of the next release :)
